Resources/config/routing_pjax.yml

```diff
@@ -147,7 +147,7 @@ admin_roleList:
         _controller: ezsystems.platformui.controller.role:listRolesAction
 
 admin_roleView:
-    path: /role/view/{roleId}
+    path: /role/view/{roleId}/{tabId}
```

Comment:
but in any case, this parameter is not necessary since the selected tab will be handle by the JS code (see https://github.com/ezsystems/PlatformUIBundle/pull/438/files#r46159286)